File: backend/models/Standings.js
import { sequelize } from "../db.js";
import { DataTypes, DATE } from "sequelize";


const Standings = sequelize.define(
    "Standings",
    {
        Team: {
            type: DataTypes.STRING,
        },
        Place: {
            type: DataTypes.INTEGER,
        },
        WinSeries: {
            type: DataTypes.INTEGER,
        },
        LossSeries: {
            type: DataTypes.INTEGER,
        },
        Streak: {
            type: DataTypes.INTEGER,
        },
        StreakDirection: {
            type: DataTypes.INTEGER,
        },
        Points: {
            type: DataTypes.INTEGER,
        }
    }
)

export default Standings;

File: ekada-dotnet/models/Standings.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ekada_dotnet.Models;

[Table("standings")]
public class Standings
{
    [Key]
    public int Id {get; set;}
    public required string LeagueName {get; set;}
    public string? Team {get; set;}
    public int Place {get; set;}
    public int WinSeries {get; set;}
    public int LossSeries {get; set;}
    public string? Streak {get; set;}
    public int Points {get; set;}

}